### PR TITLE
#166790008 List Users Functionality

### PR DESCRIPTION
**Pull request by Hervera**
#### What does this PR do?
Lists users functionality

#### Description of Task to be completed
  - add a route
  - add functionality of listing users
  - add tests
  - add a documentation

#### How should this be manually tested?
1. Run the following commands in the terminal
```
git clone https://github.com/andela/ah-92explorers-backend.git
cd ah-92explorers-backend
git checkout ft-list-users-functionality-166790008
npm install
npm start
```
2. Create a .env file and add the following snippets
```
DATABASE_URL=postgresql://dbusername:dbpassword@localhost:5432/dbname
DATABASE_TEST=postgresql://dbusername:dbpassword@localhost:5432/dbnamefortest
SECRET=whatever you want it to be
```
3. Install `sequelize` globally
```
sudo npm install -g sequelize
```
4.  Create a database using `sequelize db:create`
5. Install and setup Postman to consume `list users` endpoint
6. Run the following requests in Postman using `localhost:3000`
## Endpoint
|  Method  |  Endpoint  |  Task  |
|  --- |  --- |  ---  |
|  `GET`  |  `/api/list-users`  |  `List users functionality`  |

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#166790008](https://www.pivotaltracker.com/story/show/166790008)

#### Screenshots (if appropriate)
Screen Shot 2019-07-20 at 08 53 18

#### Checklist

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I don't have more than 3 major commits on this PR

### CODE REVIEW

File: controllers/ratings.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
+import models from '../models';
```
Comment (niyoceles) [inline controllers/ratings.js]: Remove this file ratings.js in controllers will no longer be used.

File: models/users.js [hunk 2]
```diff
@@ -22,21 +24,26 @@ export default (sequelize, DataTypes) => {
         isEmail: true
       }
     },
-    provider: { type: DataTypes.STRING },
```
Comment (niyoceles) [inline models/users.js]: Please add this  column on users table `provider: { type: DataTypes.STRING },`

**Review by niyoceles — CHANGES_REQUESTED** [review]

**Review by elemanhillary-zz — APPROVED** [review]
ready for merge

**Review by elemanhillary-zz — DISMISSED** [review]
resolve merge conflicts

**Review by niyoceles — DISMISSED** [review]

**Review by elemanhillary-zz — DISMISSED** [review]
ready for merge

**Review by IsaiahRn — DISMISSED** [review]
ready to be merged

**Review by elemanhillary-zz — DISMISSED** [review]
ready for merge

**Review by IsaiahRn — APPROVED** [review]

**Review by niyoceles — DISMISSED** [review]

**Review by IsaiahRn — CHANGES_REQUESTED** [review]
fix merge conflicts

**Review by niyoceles — APPROVED** [review]

**Review by elemanhillary-zz — APPROVED** [review]
ready for merge

**Review by IsaiahRn — APPROVED** [review]

**Pull request merged**